### PR TITLE
Update "New To Matic" to include examples on how to query data from Polygon blockchain.

### PR DESCRIPTION
**Pull request by deric-alchemy**
As discussed with Rahat Chowdhury, Polygon DevRel. Showing examples on querying basic blockchain data for new Polygon users.

Full list of changes: https://docs.google.com/document/d/1t2lOd8XEW5Bdu-3YuNl5_lv4wb-MjNaycLJw7q_VwXs/edit#

### CODE REVIEW

File: sidebars.js [hunk 1]
```diff
@@ -60,6 +60,7 @@ module.exports = {
       },
       items: [
         "develop/network-details/technical-requirements",
+        "develop/network-details/access-node-alchemy",
```
Comment (DannyS03) [inline sidebars.js:63]: need to remove in this PR as the file doesn't exist here

File: docs/home/new-to-matic.md [hunk 2]
```diff
@@ -15,6 +15,12 @@ Polygon is a scaling solution for public blockchains. Polygon PoS supports all t
 * [Ethereum + Polygon with PoS bridge](https://docs.polygon.technology/docs/develop/ethereum-polygon/pos/getting-started)
 * [Ethereum + Polygon with Plasma bridge](https://docs.polygon.technology/docs/develop/ethereum-polygon/plasma/getting-started)
 
+## Query the Blockchain
+80% of web3 developers don't need to write a single smart contract - instead, most interaction with the blockchain involves reading its current state.
```
Comment (DannyS03) [inline docs/home/new-to-matic.md:19]:
```suggestion
Most blockchain interactions involve reading its state.
```

File: docs/home/new-to-matic.md [hunk 3]
```diff
@@ -15,6 +15,12 @@ Polygon is a scaling solution for public blockchains. Polygon PoS supports all t
...
+
+If you're in that 80%, here are some examples on how to make basic requests to the blockchain.
```
Comment (DannyS03) [inline docs/home/new-to-matic.md:21]:
```suggestion
Alchemy offers a reference guide on how to make basic requests to the blockchain.
Check out their guide on [how  to query Polygon](https://docs.alchemy.com/reference/polygon-sdk-examples).
```

**Review by DannyS03 — CHANGES_REQUESTED** [review]
thank you @deric-alchemy, left a few comments.

File: sidebars.js (hunk 1 above)
Comment (DannyS03) [reply sidebars.js:63]: CI will fail as a result

Comment (deric-alchemy) [reply sidebars.js]: resolved

File: docs/home/new-to-matic.md [hunk 4]
```diff
@@ -15,6 +15,13 @@ Polygon is a scaling solution for public blockchains. Polygon PoS supports all t
 * [Ethereum + Polygon with PoS bridge](https://docs.polygon.technology/docs/develop/ethereum-polygon/pos/getting-started)
 * [Ethereum + Polygon with Plasma bridge](https://docs.polygon.technology/docs/develop/ethereum-polygon/plasma/getting-started)
 
+## Query the Blockchain
+Most blockchain interactions involve reading its state.
+
+Alchemy offers a reference guide on how to make basic requests to the blockchain.
+Check out their guide on [how  to query Polygon](https://docs.alchemy.com/reference/polygon-sdk-examples).
+* [How to Query the Polygon Blockchain](https://docs.alchemy.com/reference/polygon-sdk-examples)
```
Comment (DannyS03) [inline docs/home/new-to-matic.md:23]:
```suggestion
```

**Review by DannyS03 — APPROVED** [review]

File: docs/home/new-to-matic.md [hunk 5]
```diff
@@ -15,6 +15,12 @@ Polygon is a scaling solution for public blockchains. Polygon PoS supports all t
 * [Ethereum + Polygon with PoS bridge](https://docs.polygon.technology/docs/develop/ethereum-polygon/pos/getting-started)
 * [Ethereum + Polygon with Plasma bridge](https://docs.polygon.technology/docs/develop/ethereum-polygon/plasma/getting-started)
 
+## Query the Blockchain
```
Comment (DannyS03) [inline docs/home/new-to-matic.md:18]:
```suggestion
## Query the blockchain

```

**Review by DannyS03 — APPROVED** [review]

**Pull request merged**